depends/README.md

```diff
@@ -101,6 +101,10 @@ The following can be set when running make: `make FOO=bar`
 <dd>Don't download/build/cache libs needed to enable the wallet</dd>
 <dt>NO_UPNP</dt>
 <dd>Don't download/build/cache packages needed for enabling upnp</dd>
+<dt>ALLOW_HOST_PACKAGES</dt>
+<dd>Enables the hosts packages in case automatic download/build of dependencies
+is disabled via the NO_* options (example NO_QT etc). This is useful in case of
+cross-compilation (example: compilation for macOS on debian)</dd>
```

Comment:
Two notes to consider:
- `NO_PACKAGE` is not the only way to miss package in depends (see #18051)
- in general, building with depends is used not only for cross-compilation

```suggestion
<dd>Packages that are missed in dependencies are searched for among the host
system packages using `pkg-config`</dd>
```

Reply:
@hebasto Hopefully I got it right this time. Sorry for being dense and the delay. Appreciate your help and the good learning.